schemas/context/webpage.schema.json

```diff
@@ -5,7 +5,7 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/xdm/channels/webpage",
+  "$id": "https://ns.adobe.com/xdm/context/webpage",
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Web Page",
   "type": "object",
```

Comment:
For Description:
Replace with something like:
This is the information about the page (URL) itself.

For Section:
What is a section?  There is no relevant term in HTML or the Web - please use a term that relates, or at least describe the relationship in the description.

For name:
Is this the title of the page, as defined by the title tag?  If so, say so, if not, explain what it is.

For server:
A hostname has nothing to do with a server.  What is this? It's not used in the example.

Error page:
What is an "error page"?  Can you refer to some specification that describes what one is?

Home Page:
What is a "home page"?  Can you refer to some specification that describes what one is?